Nodejs/Setup/NodejsToolsInstaller/NodejsToolsInstaller.wxs

```diff
@@ -153,11 +154,26 @@
         <CustomAction Id="VWDExpressSetup" Property="VWDEXPRESS_PATH" ExeCommand="/setup" Execute="deferred" Return="check" Impersonate="no" />
         <CustomAction Id="VWDExpressSetup_Rollback" Property="VWDEXPRESS_PATH" ExeCommand="/setup" Execute="rollback" Return="check" Impersonate="no" />
 
+        <!-- Custom Action and properties for installer telemetry-->
+        <Property Id="VSVersion" Value="$(var.VSTargetVersion)"></Property>
+        <Property Id="NTVSVersion" Value="$(var.ReleaseVersion)"></Property>
+        <Property Id="MSIVersion" Value="$(var.MsiVersion)"></Property>
+        <Property Id="InstallStartTime" Value="Default"></Property>
+        <Binary Id="Custom_Action.CA.dll" SourceFile="..\Custom_Action\bin\$(var.Configuration)\Custom_Action.CA.dll" />
+        <CustomAction Id="LogStart" BinaryKey="Custom_Action.CA.dll" DllEntry="RecordInstallStartTime"/>
+        <CustomAction Id="LogSuccess" BinaryKey="Custom_Action.CA.dll" DllEntry="LogInstallSuccessResult"/>
+        <CustomAction Id="LogFailure" BinaryKey="Custom_Action.CA.dll" DllEntry="LogInstallErrorResult"/>
+        <CustomAction Id="LogCancel" BinaryKey="Custom_Action.CA.dll" DllEntry="LogInstallCancelResult"/>
```

Comment:
I renamed the binaries and added it to buildrelease script. Please help with signing the binaries. .CA is auto-generated for MSI to talk to custom actions.